src/config/preferences.h

```diff
@@ -176,6 +176,8 @@ typedef enum {
     PREF_URL_SAVE_CMD,
     PREF_COMPOSE_EDITOR,
     PREF_SILENCE_NON_ROSTER,
+    PREF_OUTGOING_STR,
```

Comment:
I think you should somehow add `stamp` there.
Something like `PREF_OUTGOING_STAMP`

Reply:
you can grep for PREF_INCOMING_STR and PREF_OUTGOING_STR through this PR
it's exactly the same what you mean by PREF_OUTGOING_STAMP

Reply:
I think `PREF_OUTGOING_STR` is a too general term. Since this is about stamps I think it should be reflected in the name.

Reply:
I did that.